contracts/Comptroller/Diamond/interfaces/IMarketFacet.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity 0.8.25;
 
-import { VToken } from "../../../Tokens/VTokens/VToken.sol";
+import { IVToken } from "../../../Tokens/VTokens/interfaces/IVToken.sol";
 
 interface IMarketFacet {
```

Comment:
Missing natspec for all functions in the interface?

Reply:
c974cf206a7fc31c5b03ee5f46f4d94228c62d12